tabs/src/components/app/views/Monaco.jsx

```diff
@@ -28,7 +31,7 @@ const Monaco = ( props ) => {
         wordWrap: 'on'
       }}
       onChange={onChange}
-      theme='vs'
+      theme={themeString === 'default' ? 'vs' : 'vs-dark'}
```

Comment:
very nice. so high contrast is also vs-dark? (i think that's how the web app does it too so this lgtm!)

Reply:
yuh